ui/src/authors/components/AssignAction.jsx

```diff
@@ -54,10 +50,12 @@ function AssignAction({ onAssignToAnotherAuthor, onAssign, disabled }) {
           key="assign-self"
           onClick={onSelfAssign}
         >
-          This is my paper
+          {numberOfSelected === 1 ? 'This is my paper' : 'These are my papers'}
```

Comment:
@michamos probably you missed that https://github.com/cern-sis/issues-inspire/issues/32#issuecomment-1119433603 is it ok to display different text based on the number of selected papers?

Reply:
I talked to Micha and it's ok :)